### PR TITLE
A handful of theme tweaks and improvements.

### PR DESCRIPTION
**Pull request by chriskrycho**
Some tweaks:

<details><summary>desktop</summary>

notion-desktop

</details>

<details><summary>mobile</summary>

notion-mobile

</details>
<details><summary>mobile nav</summary>

notion-mobile-nav

</details>

---

(Note that this doesn't include the full-width hero.)

### CODE REVIEW

File: _sass/notion/_alerts.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .alert.alert-warn {
     background-color: rgba(255,229,100,.3);
-    border-color: #e7c000;
+    color-primary-very-light: #e7c000;
```
Comment (dherman) [inline _sass/notion/_alerts.scss:3]: I don't understand this line... isn't `color-primary-very-light` a SASS variable, not a CSS key?

**Review by dherman — APPROVED** [review]
Would it be possible to get a dark pink color that's just a little less muddy? Maybe something slightly more "fluorescent"? (I'm bad at describing colors with English, hope that makes sense?)

Anyway I'll merge for now and we can keep iterating!

File: _sass/notion/_alerts.scss (hunk 1 above)
Comment (chriskrycho) [reply _sass/notion/_alerts.scss:3]: Umm, why, yes. That’ll teach me to use regexes and not double check what they did. (Also: CSS is tooooo forgiving.)

**Comment by chriskrycho** [comment]
I’ll have a fix for that border thing and also a color tweak (100% on board and easy to do!) this afternoon.

**Pull request merged**